### ISSUE TITLE
Unclear what current privacy setting on new tweet is

### ISSUE THREAD

**Issue by boredzo** [pcperini/Tusk---Issues#41]
## Expected Behavior
It should be clear what privacy setting is currently selected. Ideally, the privacy button should indicate it, and there should be a checkmark on the current option in the privacy options menu.

## Actual Behavior
There's a privacy button with a menu of options, but it does not reflect which option is currently in effect.

![Screenshot of the compose view in b29.](https://user-images.githubusercontent.com/49781/45118536-9b1f2180-b10d-11e8-83b7-faad832ffa98.jpg)

… I just noticed here on my Mac that there *is* a current-privacy icon, but it's such a light gray as to be hard to spot on the phone, and it's in a completely different place from the button to change the current privacy setting.

## Steps to Reproduce the Problem

  1. Begin composing a toot.
  1. Wonder what privacy it would be posted with.
  1. Tap on the privacy button to see what's currently selected.

## Specifications

  - Version: 1.0.0 b29
  - Platform: iOS
  - Subsystem: Compose view

**Comment by pzumk** [comment]
The current status is shown in the top right

**Comment by boredzo** [comment]
Yes, I mentioned that in the description, and also why it didn't help me.

**Comment by pzumk** [comment]
I'm sorry I did not see that
